tensorflow/contrib/go/graph_test.go

```diff
@@ -0,0 +1,252 @@
+package tensorflow_test
+
+import (
+	"testing"
+
+	tf "github.com/tensorflow/tensorflow/tensorflow/contrib/go"
+)
+
+func TestGraphGeneration(t *testing.T) {
+	graph := tf.NewGraph()
+	input1 := graph.Placeholder("input1", tf.DtInt32, []int64{3}, []string{})
+	input2 := graph.Placeholder("input2", tf.DtInt32, []int64{3}, []string{})
```

Comment:
Yes, I got the same impression, I added the name because on the protobuf definition all the dimensions have a name, but I think that is completely unnecessary, I'm going to remove the names :)